### PR TITLE
fixtures: fix fixtures loading

### PR DESCRIPTION
**Pull request by jrcastro2**

### CODE REVIEW

**Comment by jrcastro2** [comment]
> could you give us some context on what was the issue ?

Sure!
Basically it was loading only data from `invenio_app_rdm`, meaning that if it's empty as it's the case by default, it was not taking into account the isntance data folder.
By using `app_data_folder = Path(current_app.instance_path) / "app_data"` we make sure that we prioritize the instance app_data folder over the `invenio-app-rdm` data folder.

**Review by kpsherva — APPROVED** [review]

File: invenio_app_rdm/fixtures/__init__.py [hunk 1]
```diff
@@ -34,15 +39,18 @@ class FixturesEngine:
     def run(self):
         """Run the fixtures loading."""
         dir_ = Path(__file__).parent
-
+        app_data_folder = Path(current_app.instance_path) / "app_data"
+        app_pages_folder = Path(current_app.instance_path) / "app_data/pages"
```
Comment (slint) [inline invenio_app_rdm/fixtures/__init__.py:43]:
```suggestion
        app_pages_folder = app_data_folder / "pages"
```

Ditto

**Review by slint — APPROVED** [review]
LGTM, just reusing a variable maybe for clarity.

File: invenio_app_rdm/fixtures/__init__.py [hunk 2]
```diff
@@ -19,11 +21,14 @@ class Pages:
     def run(self, force=False):
         """Run the pages loading."""
         dir_ = Path(__file__).parent
-
+        app_data_folder = Path(current_app.instance_path) / "app_data"
+        app_pages_folder = Path(current_app.instance_path) / "app_data/pages"
```
[the remark this answers is not in the document]
Comment (jrcastro2) [reply invenio_app_rdm/fixtures/__init__.py]: RIght, thanks!

**Pull request merged**